pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -207,6 +207,25 @@ var MinimumSupportedPVCSizeByProvisionerKey = map[string]string{
 	"csi.trident.netapp.io/gcnv-flex": "1Gi",
 }
 
+// DataImportCronAccessModesByProvisionerKey defines required access modes for DataImportCron PVCs
+// Some provisioners require specific access modes for DataImportCron-created PVCs
+var DataImportCronAccessModesByProvisionerKey = map[string][]v1.PersistentVolumeAccessMode{
+	"pd.csi.storage.gke.io":           {rwo},
+	"pd.csi.storage.gke.io/hyperdisk": {rwo},
+}
+
+// DataImportCronSnapshotClassParametersByProvisionerKey defines required VolumeSnapshotClass parameters for DataImportCron.
+// Some provisioners require specific parameters in the VolumeSnapshotClass for DataImportCron snapshots.
+var DataImportCronSnapshotClassParametersByProvisionerKey = map[string]map[string]string{
+	// https://docs.cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/backup-pd-volume-snapshots#restore-snapshot
+	"pd.csi.storage.gke.io": {
+		"snapshot-type": "images",
+	},
+	"pd.csi.storage.gke.io/hyperdisk": {
+		"snapshot-type": "images",
+	},
+}
```

Comment:
To me, the fact that this degree of handling is required in CDI screams
that for that provider, there is a need for a separate storage class for cron purposes.

That provider would then represent itself internally as
```go
"pd.csi.storage.gke.io/hyperdisk-crons": {{rwo, block}},
```
And would be using the "correct" volumesnapshotclass via `storageProfile.Status.SnapshotClass`
(the special storage class would hint us which one to choose via `snapclass` parameter or something)